<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VisualizarFornecedor</class>
 <widget class="QWidget" name="VisualizarFornecedor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1324</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Visualizar Fornecedor</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Icons/mdpi.png</normaloff>../Icons/mdpi.png</iconset>
  </property>
  <widget class="QTableWidget" name="tblFornecedor">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>931</width>
     <height>535</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>133</number>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Id</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nome</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Descrição</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Cidade</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Endereço</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Bairro</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Numero</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="tblTelefone">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>0</y>
     <width>381</width>
     <height>264</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>541</width>
     <height>264</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="showGrid">
    <bool>true</bool>
   </property>
   <property name="gridStyle">
    <enum>Qt::SolidLine</enum>
   </property>
   <property name="sortingEnabled">
    <bool>false</bool>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="cornerButtonEnabled">
    <bool>true</bool>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>126</number>
   </attribute>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
   <column>
    <property name="text">
     <string>DDD</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Numero</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Referencia</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="tblEmail">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>270</y>
     <width>381</width>
     <height>264</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>541</width>
     <height>264</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="showGrid">
    <bool>true</bool>
   </property>
   <property name="gridStyle">
    <enum>Qt::SolidLine</enum>
   </property>
   <property name="sortingEnabled">
    <bool>false</bool>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="cornerButtonEnabled">
    <bool>true</bool>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>190</number>
   </attribute>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Email</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Referencia</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
